Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -383,6 +383,15 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
         setupNavigationService()
         setupVoiceController()
         setupNavigationCamera()
+        
+        if #available(iOS 12.0, *) {
```

Comment:
When https://github.com/mapbox/mapbox-navigation-ios/pull/4142 lands, there will be no need to check whether iOS 12 is available.